src/utils/command.js

```diff
@@ -69,14 +69,14 @@ class BaseCommand extends Command {
   }
 
   // Find and resolve the Netlify configuration
-  async getConfig(cwd, state, token) {
+  async getConfig(cwd, state, token, argv) {
     try {
       return await resolveConfig({
         config: argv.config,
         cwd: cwd,
         context: argv.context,
         debug: argv.debug,
-        siteId: state.get('siteId'),
+        siteId: argv.siteId || typeof argv.site === 'string' && argv.site || state.get('siteId'),
```

Comment:
Do we have any CLI flag called `siteId` instead of `site`?

Reply:
Yes, we have a `site` flag in `netlify deploy`

Reply:
We have a `site` CLI flag in other commands as well. But I was wondering about a `siteId` CLI flag?

Reply:
The other `site` flag is a boolean. That's why I'm checking for `typeof`. We're checking for both `siteId` and a string `site` flag now.

Reply:
I am wondering if `argv.siteId ||` could be removed since we don't have any `siteId` CLI flag at the moment?